casaos-toolkit/run.sh

```
@@ -0,0 +1,248 @@
+
+# Function to fix Overlay2 issues
+fix_overlay2() {
+    echo -e "${YELLOW}Fixing Docker overlay2 issues...${NC}"
+    
+    # Stop services
+    systemctl stop docker
+    systemctl stop casaos
+
+    # Backup existing Docker root
+    if [ -d "/var/lib/docker" ]; then
+        mv /var/lib/docker /var/lib/docker.bak.$(date +%Y%m%d_%H%M%S)
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)